src/content/implementations/tus-android-client.json

```diff
@@ -10,5 +10,6 @@
   "description": "Add tus <code>v1.0.0</code> resumable file uploads to <strong>Android</strong> apps.",
   "license": "MIT",
   "type": "official",
+  "subtype": "client",
```

Comment:
Having type and subtype seems a bit confusing to me because the names do not really reflect what values it might have. Maybe we can have:

```
isOfficial: boolean
type: "server" | "client"
```

for all implementations?

Reply:
This is how Nick implemented it and what it already in use in all other files. For what it's worth, TS does infer it:
<img width="1419" alt="Screenshot 2023-07-19 at 09 33 40" src="https://github.com/tus/tus.io/assets/9060226/b4e4f90b-fb7d-4179-aa86-941485b3204a">